### PR TITLE
Fix : README.md : Typos and punctuations.

### PR DESCRIPTION
**Pull request by akk312000**
There are some typos and punctuation mistakes in the README.md file.
I hope I got them all.

Thanks.

### CODE REVIEW

**Comment by Suraj-Upadhyay** [comment]
Hii @akk312000 , This patch looks good to me.

But for me to be able to merge it into master please do the following :-
1. Rebase this branch `patch-1` to MimimalOS/master. Just do `git pull origin master`
   and `git rebase master`.
2. Change the commit message to something like this : "Fix : README.md : Typos and Punctuations".

And thanks for submitting a Pull Request.
Apologies for the delayed response.

File: README.md [hunk 1]
```diff
@@ -9,22 +9,22 @@ multitasking, etc.
...
+    3. Screen and keyboard drivers.
+    4. Basic _libc_ purely for the current requirements.
+    5. Filesystem.
+    6. Basic Memory Management.
+    7. Simple Shell.
+    8. A basic text editor.
+    9. User mode.
 
-I would like to stretch my goals for :-
+I would like to stretch my goals for:-
```
Comment (Suraj-Upadhyay) [inline README.md]: I think this 'for' doesn't look good and should be something like -
"I would like to stretch my goals to implement:-"
What do you think ??

File: README.md [hunk 2]
```diff
@@ -9,22 +9,22 @@ multitasking, etc.
...
 
-	1. Network Stack.
-	2. Basic Interpreter.
+    1. Network Stack.
```
Comment (Suraj-Upadhyay) [inline README.md]: It should be something like "The Network Stack".

File: README.md [hunk 3]
```diff
@@ -9,22 +9,22 @@ multitasking, etc.
...
+    2. Basic Interpreter.
```
Comment (Suraj-Upadhyay) [inline README.md]: "A basic interpreter". The same thing you did above.

File: README.md [hunk 4]
```diff
@@ -51,11 +51,11 @@ As said earlier, once this project leaves the examiners room, it will be open
 for all.
 
 P.S. If you do want to contribute to this project, you should open an issue asking to contribute
-and hopefully we can give you a feature to implement which is not in the above list.
+and hopefully, we can give you a feature to implement which is not in the above list.
 
 ## Post Scriptum
 
-Apologies, if this repository is not up to the standards of Open Source Software Community.
+Apologies, if this repository is not up to the standards of the Open Source Software Community.
```
Comment (Suraj-Upadhyay) [inline README.md]: I don't think we need a "the" here.
WDYT ??

File: README.md (hunk 1 above)
Comment (akk312000) [reply README.md]: Sir, the latter looks more clear to me too, so I think we should modify it!

File: README.md (hunk 2 above)
Comment (akk312000) [reply README.md]: Yes sir , it looks good to me!

File: README.md (hunk 4 above)
Comment (akk312000) [reply README.md]: Sir, I think the same, I would omit this!

**Comment by Suraj-Upadhyay** [comment]
Just Do : 
1. `git checkout master`
2. `git pull origin master`
3. `git checkout patch-1`
4. `git rebase master`
5. `git rebase -i HEAD~2`
6.  Squash the two commits you have made into one.
7.  Now reword your single commit into the someting like
    Fix : README.md : Typo and punctuations.
8. `git push --force upstream patch-1`

**Comment by Suraj-Upadhyay** [comment]
And if you want to contribute to this project in the future, please change to the original repository[1] and send pull requests and/or submit issues over there.

[1] :- For that you don't have to delete and reclone, you just have to : 
1. `git remote remove origin`
2. `git remote add origin https://github.com/MrSquanchee/MinimalOS.git`.
3. `git pull origin master`. // You know to update your repostiory

Thanks :)

**Review by Suraj-Upadhyay — APPROVED** [review]
Looks Good To Me. Merging it soon.

**Pull request merged**